No Message

### Description

In the file About.cshtml under the Pages folder, there is a reference to Message, but there is no Message property in About.cshtml.cs.  I get the error CS1061 for About.cshtml, line 7.

[Enter feedback here]


### Page URL

https://learn.microsoft.com/en-us/aspnet/core/fundamentals/logging/?view=aspnetcore-9.0

### Content source URL

https://github.com/dotnet/AspNetCore.Docs/blob/main/aspnetcore/fundamentals/logging/index.md

### Document ID

726e3bf1-f367-d733-8933-bccc04da0e16

### Platform Id

07f49229-07df-50f0-e102-82d349968970

### Article author

@tdykstra

### Metadata

* ID: 7184a308-a254-9350-a5cd-5bced1f369ae
* PlatformId: 07f49229-07df-50f0-e102-82d349968970 
* Service: **aspnet-core**
* Sub-service: **fundamentals**

[Related Issues](https://github.com/dotnet/AspNetCore.Docs/issues?q=is%3Aissue+is%3Aopen+726e3bf1-f367-d733-8933-bccc04da0e16)

--- a/aspnetcore/fundamentals/logging/index/samples/6.x/TodoApiDTO/Pages/About.cshtml
+++ b/aspnetcore/fundamentals/logging/index/samples/6.x/TodoApiDTO/Pages/About.cshtml
@@ -4,5 +4,3 @@
     ViewData["Title"] = "About page";
 }
 <h2>@ViewData["Title"]</h2>
-<h3>@Model.Message</h3>
-

